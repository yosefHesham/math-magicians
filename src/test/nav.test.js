import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Navbar from '../components/navbar';

describe('Test for navbar inside the router', () => {
  test('Navbar should render correctly', () => {
    const nav = renderer
      .create(<Router><Navbar /></Router>)
      .toJSON();
    expect(nav).toMatchSnapshot();
  });
});