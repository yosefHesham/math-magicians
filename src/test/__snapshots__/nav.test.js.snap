// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for navbar inside the router Navbar should render correctly 1`] = `
<nav
  className="navBar"
>
  <h1>
     Math Magicians 
  </h1>
  <ul>
    <li>
      <a
        href="/"
        onClick={[Function]}
      >
        Home
         
        |
      </a>
    </li>
    <li>
      <a
        href="/calc"
        onClick={[Function]}
      >
        Calculator
         
        |
      </a>
    </li>
    <li>
      <a
        href="/quote"
        onClick={[Function]}
      >
        Quote
         
      </a>
    </li>
  </ul>
</nav>
`;
